src/code/z_demo.c

```diff
@@ -178,49 +182,57 @@ void Cutscene_Command_Misc(PlayState* play, CutsceneContext* csCtx, CsCmdBase* c
                 play->envCtx.unk_22 = play->envCtx.unk_24;
             }
             break;
-        case 0x5:
-            if (isStartFrame && (csCtx->state != CS_STATE_4)) {
-                csCtx->state = CS_STATE_3;
+
+        case CS_MISC_STOP_CUTSCENE:
+            if (isFirstFrame && (csCtx->state != CS_STATE_RUN_UNSTOPPABLE)) {
+                csCtx->state = CS_STATE_STOP;
             }
             break;
-        case 0x7:
-            if (isStartFrame) {
+
+        case CS_MISC_SHOW_TITLE_CARD:
+            if (isFirstFrame) {
                 loadedScene = play->loadedScene;
                 if (loadedScene->titleTextId != 0) {
                     func_80151A68(play, loadedScene->titleTextId);
                 }
             }
             break;
-        case 0x8:
+
+        case CS_MISC_MEDIUM_QUAKE_START:
             func_8019F128(NA_SE_EV_EARTHQUAKE_LAST - SFX_FLAG);
-            if (isStartFrame) {
```

Comment:
may be worth just calling this "earthquake" given the sound it plays. "medium quake" seems like its just the camera quake effect